vms/evm/destination_client.go

```diff
@@ -134,32 +134,24 @@ func (tdc *destinationClient) SendTx(signedMessage *avalancheWarp.Message,
 		return err
 	}
 
-	// Pack the signed message to be delivered in the storage slots.
-	// The predicate bytes are packed with a delimiter of 0xff.
-	predicateBytes := predicateutils.PackPredicate(signedMessage.Bytes())
-
 	to := common.HexToAddress(toAddress)
-
 	gasFeeCap := baseFee.Mul(baseFee, big.NewInt(BaseFeeFactor))
 	gasFeeCap.Add(gasFeeCap, big.NewInt(MaxPriorityFeePerGas))
 
 	// Construct the actual transaction to broadcast on the destination chain
-	tx := types.NewTx(&types.DynamicFeeTx{
-		ChainID:   destinationChainIDBigInt,
-		Nonce:     tdc.currentNonce,
-		To:        &to,
-		Gas:       gasLimit,
-		GasFeeCap: gasFeeCap,
-		GasTipCap: gasTipCap,
-		Value:     big.NewInt(0),
-		Data:      callData,
-		AccessList: types.AccessList{
-			{
-				Address:     warp.ContractAddress,
-				StorageKeys: predicateutils.BytesToHashSlice(predicateBytes),
-			},
-		},
-	})
+	tx := predicateutils.NewPredicateTx(
```

Comment:
the `predicateutils.NewPredicateTx` just helps form a predicate tx that packs the predicate bytes, and forms the access list with the predicate address. Will perform the same as we previously had, but I think it's good to use the `predicateutils` in case of any changes on subnet-evm side.